examples/client/mir_shell_demo.cpp

```diff
@@ -67,11 +68,21 @@ void parse_options(int argc, char* argv[])
         case 0:
             if (option_main_width == long_options[option_index].name)
             {
-                main_width = atoi(optarg);
+                int32_t parsed_value;
```

Comment:
You haven't initialized the value here (and below) like in the third case. Even though this value is only accessed on success it might be safer to set it to zero.